dist/pathfora.js

```diff
@@ -2302,6 +2308,28 @@
      */
     this.version = '0.0.04';
 
+    /**
+     * @public
+     * @description callbacks to execute once segments load
+     */
+    this.callbacks = [];
+
+    /**
+     * @public
+     * @description Add callbacks to execute once segments load
+     */
+    this.addCallback = function (cb) {
+      if (context.lio) {
+        if (!context.lio.segmentsCbCalled) {
```

Comment:
Oh sweet, I didn't know there was a variable for this already. I will fix in both PRs. :)